css/build.css

```diff
@@ -163,3 +163,8 @@ div[data-state="touchID"] button {
 	vertical-align: top;
 	color: #e9376b;
 }
+
+/* Fix component view when it is placed in the container */
+[data-widget-package="com.fliplet.lock"] {
+  width: 100%;
```

Comment:
I'm not going to approve this until @hcarneiro answers the question at https://github.com/Fliplet/fliplet-widget-fliplet-login/pull/50/files#r386035704